### DISCUSSION TITLE
NAT events logging

### DISCUSSION THREAD

**Discussion by rpravm** [akvorado/akvorado#388] [Q&A]
I understand that akvorado is mainly designed for traffic analysis, but it is build for ISP needs in mind.

We have a need of collecting, enriching, storing and searching through NAT events when needed.

Does this feature matches entire design model of akvorado?

Maybe there is some way for merging this information with main flows data? 
In that way this info just extends main flows table with NAT info.

Below example of event sending from NAT router:

<img width="715" alt="image" src="https://user-images.githubusercontent.com/35486468/211398976-37469660-122d-4766-8a8b-2b00ab5f48c6.png">

**Comment by vincentbernat** [comment]
Isn't your request covered by https://github.com/akvorado/akvorado/issues/211? In this case, it should happen "soon".

**Reply by rpravm** [reply]
Yes and no. We have this functionality on different hardware and it is sending different flows.
Someone have this in one box.

It's a different approaches but both are widely used in ISP networks.
